guides/Benchmarking_shuffle_guide.md

```diff
@@ -31,34 +43,57 @@ Creating a new shuffle with the same train/test split as an existing one:
 4. Click 'View existing shuffles':
-    - This is used to view the indices of shuffles that have been created for a project, in order to determine which index is available to assign to a new shuffle.
+    - This is used to view the indices of shuffles that have been created for a project,
+in order to determine which index is available to assign to a new shuffle.
     - The elements described in this window are:
         - train_fraction: The fraction of the dataset used for training.
         - index: The index of the shuffle.
-        - split: The data split for the shuffle. The integer value on its own does not hold any meaning, but this "split" value indicates which shuffles have the same split (as their results can then be compared)
+        - split: The data split for the shuffle. The integer value on its own does not
+hold any meaning, but this "split" value indicates which shuffles have the same split 
+(as their results can then be compared)
         - engine: Whether it's a PyTorch or TensorFlow shuffle
 ![view_existing_sh](<assets/Screenshot 2024-07-29 at 17.10.29.png>)
-5. Choose the index of the training shuffle you want to replicate. Let's assume we want to replicate the train-test split from OpenfieldOct30-trainset95shuffle3, in which split: 3 in this case, we insert in the *'From shuffle'* menu
+5. Choose the index of the training shuffle you want to replicate. Let's assume we want
+to replicate the train-test split from OpenfieldOct30-trainset95shuffle3, in which
+split: 3 in this case, we insert in the *'From shuffle'* menu
 ![choose_existing_index](<assets/Screenshot 2024-07-29 at 17.12.17.png>)
-6. In order to create this new dataset, set the shuffle option to an un-used shuffle (here 4)
+6. In order to create this new dataset, set the shuffle option to an un-used shuffle
+(here 4)
 ![choose_new_index](<assets/Screenshot 2024-07-29 at 17.36.44.png>)
-7. Click *'Create training dataset'* and move on to *'train network'*. Shuffle should be set to the new shuffle you entered at the previous step (in this case, 4)
+7. Click *'Create training dataset'* and move on to *'train network'*. Shuffle should be 
+set to the new shuffle you entered at the previous step (in this case, 4)
 ![create_from_existing](<assets/Screenshot 2024-07-29 at 17.47.10.png>)
-8. If you wish to keep the training attributes identical to your initial TensorFlow model, specifications of the original model can be found in the model folder dlc-models-pytorch > iteration folder (here 0) > shuffle (in this case 3) > train > pytorch_config.yaml. Here all parameters of the original model can be found.
+8. If you wish to keep the training attributes identical to your initial TensorFlow
+model, specifications of the original model can be found in the model folder
+dlc-models-pytorch > iteration folder (here 0) > shuffle (in this case 3) > train > 
+pytorch_config.yaml. Here all parameters of the original model can be found.
+
```

Comment:
I would edit this a bit

For TensorFlow models, the specifications for the net type and model configuration can be found in `dlc-models > iteration folder (here 0) > shuffle (in this case 3) > train > pose_config.yaml` (it's for PyTorch models that it's found in `pytorch_config.yaml`)

> If you wish to keep the training attributes identical to your initial TensorFlow

This isn't really possible: the only thing you can keep "identical" is the net type (such as using a ResNet 50). Otherwise the image augmentation, ... changed between PyTorch and TensorFlow, so really the net type is the only thing you can keep.

Reply:
How about this:

8. To view/edit the specifications of the model you created, you can go to `pytoch_config.yaml` file at:
```
dlc-project
|
|___ dlc-models-pytorch
     |__ iterationX
         |__ shuffleX
             |__ pytorch_config.yaml
```

Reply:
Yeah I think that would be great! I think we should update to use iterationX and shuffleX in the `Important files & folders` section too

Reply:
Right, just updated it